kancolle_auto.sikuli/config_reader.sikuli/config_reader.py

```diff
@@ -60,7 +60,9 @@ def get_config(settings, sleep_cycle):
     # 'Combat' section
     if config.getboolean('Combat', 'Enabled'):
         settings['combat_enabled'] = True
-        settings['combat_fleetcomp'] = config.getint('Combat', 'FleetComp')
+        settings['combat_fleetcomps'] = config.get('Combat', 'FleetComps').replace('', '').split(',')
```

Comment:
This splits into a list of strings. You cast to `int` on `pop` in `kancolle_auto.py` but config validation should be done in `config_reader`. Either pass into list comprehension:

```python
settings['combat_fleetcomps'] = [int(x) for x in config.get('Combat', 'FleetComps').replace('', '').split(',')]
```
or `map()`:
```python
settings['combat_fleetcomps'] = map(int, config.get('Combat', 'FleetComps').replace('', '').split(','))
```

Should also pass this step into a `try`/`except ValueError` block to catch non-int values in the config (another reason to move the cast here, instead of in `kancolle_auto.py`.